canonical-websites/www.canonical.com
# Replace careers sites from Taleo to Greenhouse

# Done

* replace taleo with greenhouse for careers

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [careers/all-vacancies](http://0.0.0.0:8002/careers/all-vacancies)

## Issue / Card

Fixes #226
